manifests/webclient.pp

```diff
@@ -31,6 +32,23 @@
     content => template('i2b2/webclient_config_data.js.erb'),
   }
 
+  file { "$webclient_dir/proxy.php" :
+    ensure  => file,
+    content => template('i2b2/proxy.php.erb'),
+  }
+
+  file { "$intermediate_dir/css_declarations.xml":
+    ensure  => file,
+    content => template('i2b2/css_declarations.erb'),
+    require => Exec[ "extract-$webclient_zip" ],
+  }
+  ~>
+  exec { 'insert-css' :
+    cwd     => $webclient_dir,
+    command => "sed -i -e '/<\\/head>/r $intermediate_dir/css_declarations.xml' -e //N default.htm",
```

Comment:
I know. I thought about that but I could not find an easy way to fix this. It would require removing the old ones. Not knowing which they are I would need to re-grab the original file. I am not a puppet guru so you may have some suggestions. On the other hand, I do not think it is that important that we put a lot of time in.